lib/src/runner/configuration.dart

```diff
@@ -27,6 +28,19 @@ import 'configuration/values.dart';
 /// The key used to look up [Configuration.current] in a zone.
 final _currentKey = new Object();
 
+/// The default line length for output.
+final int _defaultLineLength = () {
```

Comment:
This should go in `lib/src/configuration/values.dart`.

Reply:
done